### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by jgleissner**
Include GCE in supported platforms.
Adjust documentation to reflect behavior changes, namely routing policies also for main interface and no removal of addresses it didn't add.
Add info about systemd timer.
Add info about CLOUD_NETCONFIG_MANAGE default variable.
Add more detail about mode of operation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,35 +2,65 @@ cloud-netconfig
...
-running `udevadm control -R`. Afterwards add **cloud-netconfig** to the
-variable **NETCONFIG__MODULES__ORDER** in `/etc/sysconfig/network/config` and
-restart networking (`systemctl restart wicked.serice` on SLE 12 or recent
-openSUSE distributions).
+If you are installing from source, run as root `make install-ec2`, `make
+install-azure`, or `make install-gce` depending on your platform. Then reload
+the udev rules by running `udevadm control -R`. Afterwards add
+**cloud-netconfig** to the variable **NETCONFIG__MODULES__ORDER** in
+`/etc/sysconfig/network/config` and restart networking (`systemctl restart
+wicked.serice` on SUSE Linux Enterprise Server or openSUSE distributions).  On
```
Comment (aosthof) [inline README.md:16]: Typo: wicked.ser_v_ice

File: README.md [hunk 2]
```diff
@@ -2,35 +2,65 @@ cloud-netconfig
...
+- Set up unconfigured interfaces
+
+For any network interface that does not have an associated configuration file
+in `/etc/sysconfig/network`, a DHCP based configuration will be generated and
+`ifup` will be called, which triggers interface configuration through `wicked`.
+
+- Apply secondary IPv4 addresses
+
+For all interfaces managed by **cloud-netconfig**, it will look up secondary
+IPv4 addresses from the framework's metadta server and configure them on the
```
Comment (aosthof) [inline README.md:34]: Typo: metad_a_ta

File: README.md [hunk 3]
```diff
@@ -39,4 +69,7 @@ that it will not overwrite existing interface configurations. This allows to
 use specific interface configurations. **cloud-netconfig** will still set up
 secondary IP addresses and routing policies. If you do not want that, set the
 variable **CLOUD__NETCONFIG__MANAGE** to **no** in the `ifcfg` file in
-`/etc/sysconfig/network`.
+`/etc/sysconfig/network` to disable it for the associated interface. You can
+also change the default value of **CLOUD__NETCONFIG__MANAGE** for in
```
Comment (aosthof) [inline README.md:73]: Obsolete word "for".

**Review by aosthof — CHANGES_REQUESTED** [review]
Easy to read and understandable, LGTM aside from the 3 typos.

File: README.md (hunk 3 above)
Comment (jgleissner) [reply README.md]: Thanks for proofreading, typos fixed.

**Pull request merged**